src/utils.js

```diff
@@ -32,11 +32,11 @@ const download = async (url, path) => {
 };
 
 const getLatestVersion = async () => {
-  const endpoint = 'https://updater.dvc.org';
+  const endpoint = 'https://api.github.com/repos/iterative/dvc/releases/latest';
```

Comment:
`updater.dvc.org` is updated only after all the packages are available (in the past, we have tried to add more delay so that users are not notified immediately of the updates).

When using GitHub's API, there is a delay between creating a release and "building and uploading" the assets to that release.

Reply:
> updater.dvc.org is updated only after all the packages are available

Where is this logic implemented? I don't see it in https://github.com/iterative/lambda-latest-version/blob/master/.github/workflows/update.yaml .

As far as I can see, the only download check for packages happens in https://github.com/iterative/dvc.org/blob/master/.github/workflows/download-link-check-deploy.yml

So `updater` (if working) could also be in a status where the packages are not available?

Reply:
> Where is this logic implemented? I don't see it in [https://github.com/iterative/lambda-latest-version/blob/master/.github/workflows/update.yaml](https://github.com/iterative/lambda-latest-version/blob/master/.github/workflows/update.yaml?rgh-link-date=2022-04-18T11%3A24%3A45Z) .

@daavoo, there we only check for the latest update once every day, so the packages are already available at this point (it only takes ~7 mins max).Also, it only comes into effect after we merge the PR, so we ourselves maintain that delay.